docs/antrea-network-policy.md

```diff
@@ -605,16 +605,15 @@ be referenced in ACNP in place of stand-alone `podSelector` and/or `namespaceSel
 In addition, ClusterGroup also supports Pod grouping by `serviceReference`. ClusterGroup
 specified by `serviceReference` will contain the same Pod members that are currently
 selected by the Service's selector.
+
 ClusterGroups allow admins to separate the concern of grouping of workloads from
 the security aspect of Antrea-native policies.
 It adds another level of indirection allowing users to update group membership
 without having to update individual policy rules.
+
 In addition to specifying label selectors to group workloads, admins can create a
-ClusterGroup to share IPBlocks.
-An `ipBlock` selector may not be specified with a `podSelector` and `namespaceSelector`,
-i.e. a single ClusterGroup can either group workloads or share IPBlocks.
-A ClusterGroup is cluster scoped resource and therefore can only be set in an Antrea
-ClusterNetworkPolicy's `appliedTo` and `to`/`from` peers.
+ClusterGroup to share IPBlocks between ACNPs. Furthermore, the `childGroups` field
```

Comment:
Do you think better to list all supported group criteria after the first paragraph?

E.g. after "A ClusterGroup (CG) CRD is a specification of how workloads are grouped together, It allows admins to group Pods using traditional label selectors, which can then be referenced in ACNP in place of stand-alone `podSelector` and/or `namespaceSelector`.", we can say "ClusterGroup supports the following ways to select endpoints...".

Reply:
Updated